yarn.lock

```diff
@@ -2097,6 +2097,28 @@
     prop-types "^15.8.1"
     react-is "^18.2.0"
 
+"@mui/x-license@7.1.1":
```

Comment:
Why was this added to the lockfile?

Reply:
good question - no idea. I can remove it tho.

Reply:
done and done

Reply:
hmm removing it fails the CI - it complains that there are unstaged changes :/